internal/featuresupport/features_olm_operators_test.go

```diff
@@ -236,4 +236,85 @@ var _ = Describe("V2ListFeatureSupportLevels API", func() {
 		Entry("on S390x is    NOT supported", []string{"4.11", "4.13", "4.14", "4.21"}, models.ClusterCPUArchitectureS390x, false),
 		Entry("on ppc64le is	NOT supported", []string{"4.11", "4.13", "4.14", "4.21"}, models.ClusterCPUArchitecturePpc64le, false),
 	)
+
+	Context("Test MTV feature", func() {
+		It("Validate MTV on CPU arch", func() {
+			supportedCpuArch := []string{
+				models.ClusterCPUArchitectureArm64,
+				models.ClusterCPUArchitectureMulti,
+				models.ClusterCPUArchitectureX8664,
+			}
+			notSupportedCpuArch := []string{
+				models.ClusterCPUArchitectureS390x,
+				models.ClusterCPUArchitecturePpc64le,
+			}
+			for _, arch := range supportedCpuArch {
+				Expect(IsFeatureAvailable(models.FeatureSupportLevelIDMTV, "4.14", swag.String(arch))).To(BeTrue())
+			}
+			for _, arch := range notSupportedCpuArch {
+				Expect(IsFeatureAvailable(models.FeatureSupportLevelIDMTV, "4.14", swag.String(arch))).To(BeFalse())
+			}
+		})
+		It("Validate Feature Support for MTV", func() {
```

Comment:
@eifrach For this test, I think it's make more sense to use the one use for CNV because these two are linked.
https://github.com/openshift/assisted-service/blob/bc9d39570712e6da2a8505fff80c696e588f8ebf/internal/featuresupport/features_olm_operators_test.go#L224

WDYT?

Reply:
I see you posted the same link. nvm